### DISCUSSION TITLE
add-on install, but does not appear to activate

### DISCUSSION THREAD

**Discussion by Daniel-Vitor-infnet** [AlwarrenSidh/ArmAToolbox#60] [Q&A]
I've done the whole process of downloading the zip file, going to Preferences, Add-on and installing, but it doesn't appear to activate. In the console, it says that it went to Blender's AppData folder, and I checked that it really is there, but there's no way to activate it. I opened the text editor and ran the script, and it gave the error that BITxtWriter was not found/executed. I'll send you all the screenshots.
![imagem_2024-05-28_181533379](https://github.com/AlwarrenSidh/ArmAToolbox/assets/147751321/12e2405d-c088-4cb9-bb0b-97975575e64d)
![imagem_2024-05-28_181547534](https://github.com/AlwarrenSidh/ArmAToolbox/assets/147751321/3e05ca31-35f8-4ad5-ab32-f7e98ec98bde)
![imagem_2024-05-28_181604181](https://github.com/AlwarrenSidh/ArmAToolbox/assets/147751321/21cec64c-12e0-4acd-ac98-13a401ca01d1)

**Comment by AlwarrenSidh** [comment]
Just to make sure, you did download a release archive, NOT the "download as ZIP" from the repository?

**Reply by Daniel-Vitor-infnet** [reply]
I downloaded ArmaToolbox.zip

![image](https://github.com/AlwarrenSidh/ArmAToolbox/assets/147751321/3fb46e4d-1f51-462b-a89f-8fac023f5379)

**Comment by AlwarrenSidh** [comment]
OK to be honest, I have no idea. This looks like something fundamental. Try to uninstall the toolbox and reinstall. Failing that, try to uninstall Blender and reinstall that too... Other than that, I am at a loss, it must be something on your side since for everyone else it works, or so it seems.

**Comment by Daniel-Vitor-infnet** [comment]
I'll send the link to how I did it: https://www.youtube.com/watch?v=WCr08m_9haE

**Comment by AlwarrenSidh** [comment]
You shouldn't unpack the addon but rather when you press install, browse to the ZIP archive and select that. That should be all you have to do.
